tsconfig.json

```diff
@@ -0,0 +1,39 @@
+{
```

Comment:
This seems like a lot of stuff to put at the root level of the monorepo. Maybe it all makes sense, but I see for example that calypso's repo doesn't have a root-level tsconfig at all, and [their packages/tsconfig.json](https://github.com/Automattic/wp-calypso/blob/f529d910cd39e76a350ad6dfa95ef3aa15877189/packages/tsconfig.json) is mostly just a list of references.

Reply:
I have moved the file to `projects/` and updated the paths. Thank you.

Reply:
Moving it to `projects/` throws this error `Project directories should not contain normal files.`
https://github.com/Automattic/jetpack/runs/5672027356?check_suite_focus=true